packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-amount/index.hbs

```diff
@@ -47,6 +47,7 @@
             @symbol={{this.currentTokenSymbol}}
             @invalid={{this.isInvalid}}
             @errorMessage={{this.validationMessage}}
+            @disabled={{or @frozen (eq this.amountCtaState 'memorialized')}}
```

Comment:
Disables the amount when the amount has been used for withdrawal operations later on: 

![image](https://user-images.githubusercontent.com/273660/134667622-807827ba-cd6b-42d6-b6de-acd1e246c772.png)